lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -611,6 +614,23 @@ postExternalTransaction ctx (PostExternalTransactionData load) = do
     tx <- liftHandler $ W.submitExternalTx @ctx @t @k ctx load
     return $ ApiTxId (ApiT (txId @t tx))
 
+deleteTransaction
+    :: forall ctx s t k.
+        ( s ~ SeqState t
```

Comment:
I think `s ~ SeqState t` is superfluous here.

Reply:
indeed, fixed